### PR TITLE
Add information about CVE-2016-4451

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ The policy of the project is to treat all newly reported issues as private, and
 
 All security advisories made for Foreman are listed below with their corresponding [CVE identifier](http://cve.mitre.org/).
 
+* [CVE-2016-4451: Privileges escalation through Organization and Locations API(security.html#2016-4451)
```
Comment (domcleal) [inline security.md]: missing close `]`

File: security.md [hunk 2]
```diff
@@ -57,6 +58,16 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2016-4451"></a>CVE-2016-4451: Privileges escalation through Organization and Locations API
```
Comment (domcleal) [inline security.md]: Nitpick, "Privilege" (singular) I think

File: security.md [hunk 3]
```diff
@@ -57,6 +58,16 @@ All security advisories made for Foreman are listed below with their correspondi
...
+
+When accessing Foreman as a user limited to specific organization, if users know other organization id and have unlimited filters they can access/modify other organization data. They just have to set the id as API parameter.
+
+*Mitigation:* make sure you don't have filters not restricted to organizations or locations when you limit user by assigning him particular organization or location.
+
+* Affects Foreman 1.7 and higher
+* Fix released in Foreman 1.11.3
```
Comment (domcleal) [inline security.md]: due to be released, or planned for

File: security.md [hunk 4]
```diff
@@ -57,6 +58,16 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2016-4451"></a>CVE-2016-4451: Privileges escalation through Organization and Locations API
+
+When accessing Foreman as a user limited to specific organization, if users know other organization id and have unlimited filters they can access/modify other organization data. They just have to set the id as API parameter.
+
+*Mitigation:* make sure you don't have filters not restricted to organizations or locations when you limit user by assigning him particular organization or location.
```
Comment (domcleal) [inline security.md]: double negative, "make sure you have filters restricted to"

File: security.md (hunk 3 above)
Comment (ares) [inline security.md]: sure, I expect merging this when the PR is merged, I just wanted to be ready... or do you want to put this on web with "TBD"?

Comment (domcleal) [inline security.md]: I normally put this up immediately so the information's available and then change the wording on release.

File: security.md (hunk 4 above)
Comment (ares) [inline security.md]: I guess :notes:  "we don't need no education" confused me :-)

File: security.md (hunk 3 above)
Comment (ares) [inline security.md]: ok, makes sense

**Comment by ares** [comment]
@domcleal - fixed, could you please take another look?

**Comment by domcleal** [comment]
Thanks @ares!

**Pull request merged**